### PR TITLE
Log response of skipped promos

### PR DESCRIPTION
**Pull request by dwick**
:eyeglasses: @bsimpson63

### CODE REVIEW

File: reddit_adzerk/adzerkpromote.py [hunk 1]
```diff
@@ -903,6 +903,7 @@ def POST_request_promo(self, site, srnames, is_mobile_web, platform, loid, is_re
             return responsive(w.render(), space_compress=True)
         else:
             g.stats.simple_event('adzerk.request.skip_promo')
+            g.log.info("skipping promo: %s" % str(response))
```
Comment (zeantsoi) [inline reddit_adzerk/adzerkpromote.py]: `response.__dict__`?

Comment (bsimpson63) [inline reddit_adzerk/adzerkpromote.py]: how big is the response? might be nice to just log the campaign id that got skipped?

**Comment by dwick** [comment]
:haircut:

File: reddit_adzerk/adzerkpromote.py [hunk 2]
```diff
@@ -904,6 +905,10 @@ def POST_request_promo(self, site, srnames, is_mobile_web, platform, loid, is_re
         else:
             g.stats.simple_event('adzerk.request.skip_promo')
 
+            # TEMP: Log 1% of promo skips.
+            if random.random() <= 0.01:
+                g.log.info("skipping promo: %s" % response.campaign)
```
Comment (bsimpson63) [inline reddit_adzerk/adzerkpromote.py]: response.campaign is a valid attribute?

Comment (zeantsoi) [inline reddit_adzerk/adzerkpromote.py]: Logging just one percent... maybe just print the whole object?

**Comment by bsimpson63** [comment]
:fish:

**Pull request closed without merging**